test/e2e/framework/node/resource.go

```diff
@@ -561,6 +561,31 @@ func GetClusterZones(c clientset.Interface) (sets.String, error) {
 	return zones, nil
 }
 
+// GetSchedulableClusterZones returns the values of zone label collected from all nodes which are schedulable.
+func GetSchedulableClusterZones(c clientset.Interface) (sets.String, error) {
```

Comment:
I have added another function instead of touching existing `GetClusterZones` which is being used at multiple places.